larray_editor/editor.py

```diff
@@ -665,6 +674,33 @@ def open_tutorial(self):
     def open_api_documentation(self):
         QDesktopServices.openUrl(QUrl("http://larray.readthedocs.io/en/stable/api.html"))
 
+    def about(self):
+        """About Editor"""
+        versions = get_versions()
+        message = \
+            """<b>LArray Editor {editor}</b>
+            <br>The Graphical User Interface for LArray. This project aims to provide an intuitive tool to 
+            visualize and manipulate labelled N-dimensional arrays. Licensed under the terms of the 
+            GNU GENERAL PUBLIC LICENSE Version 3.
+            <p>Created by Gaëtan de Menten.
```

Comment:
Thanks for the mention, I really appreciate that, but I would rather have either a list of all contributors or no name at all. For practical reasons, I guess no name at all is easier :).